source/help/getting-started/creating-an-account.rst

```diff
@@ -0,0 +1,53 @@
+=================================================
+Creating an account and signing in the first time
+=================================================
+
+It's a relatively straight forward process to create an account on Mattermost and start communicating with other people. You enter some basic information on the account creation page then click **Create Account**.
+
+.. Important::
+  - The account creation process might be different than described here, depending on the policies of your organization.
+  - If your administrator sent you instructions, use those instructions.
+  - If you are not able to create an account using these instructions, contact your administrator.
+
+On the account creation page, you must enter the following information:
+
+email address
+  Enter the address that you want to use for signing in, receiving notifications, resetting passwords, and receiving your account confirmation email.
+user name
+  Enter the text that you want displayed in Mattermost. This is the name that people will know you by. If enabled by your administrator, you can also use this name to sign in.
+password
+  Enter a password that is at least 5 characters long. Your administrator might configure more restrictive requirements, such as the use of numbers, upper and lower case, and special characters. If the password that you enter does not meet one or more of the minimum requirements, you see a message that tells you what they are. You can then modify your password to meet the requirements.
+
+To create an account
+====================
+
+1. If you received a link in an email or via another message service, click the link. The account creation page opens.
+2. If you don't have a direct link:
+
+  a) Navigate to the Mattermost sign-in page. You can get the address of the sign-in page from your administrator.
+  b) Click **Create one now**. The account creation page opens.
+  
+3. Enter the following information:
+
+  * email address
+  * user name
+  * password
+  
+4. Click **Create Account**. The team selection page opens.
+5. Click the team that you want to enter. The Mattermost tutorial opens.
+6. Click **Next** to page through the tutorial. 
+7. On the last page of the tutorial, click **Next**. The all-members channel opens. By default, the all-members channel is called *Town Square*, but your administrator might have renamed it.
+
+What to do next
```

Comment:
Propose this section is titled "Updating your Profile" and directly includes the contents of the updating-your-profile.rst doc